maquetado/index.html

```diff
@@ -8,6 +8,13 @@
     <link rel="stylesheet" type="text/css" href="application.css" />
   </head>
   <body>
-    <h1 class="title">Hello World</h1>
+      <main class="main-project">
+          <section class="section-project">
+              <div class="main-title-container">
+                  <img src="../assets/triangle.svg" alt="triangle-wolox" class="main-title-img">
+                  <h1 class="main-title">Front-End Tools</h1>
+              </div>
```

Comment:
Remember to use as less html code as possible for clarity and reusability. In this case, try just to use just the `h1` element and apply the image as a decorator using only CSS (hint: you can use the `::before` selector), that way you will have a single reusable html tag (the `h1`) that contains the text, and manage the decorative part of the text via CSS only.  If you have any doubt about this ask me.